source/evp_fetch.c

```diff
@@ -23,11 +23,45 @@
 #include <openssl/core_names.h>
 #include "perflib/perflib.h"
 
+/*
+ * Enable program flag only when version is 3.5 or later
+ */
+#if OPENSSL_VERSION_MAJOR > 3 || \
+    (OPENSSL_VERSION_MAJOR == 3 && OPENSSL_VERSION_MINOR >= 5)
+# define OPENSSL_DO_PQ
+#endif
+
```

Comment:
I was talking about it on stand up yesterday. I usually build perftools for the recent version and than use LD_LIBRARY_PATH to point runtime linker to location where libcrypto.so&libssl.so are found. So basically this type of check would be better to implement as runtime check.

however runtime check is just extra effort because the EVP_fetch() will do the right thing anwawy. If we will be running perftool with the recent library the EVP_fetch() will find the PQ algorithm. If we will run it older library the EVP_fetch() will return nothing and the code already handles that (`kem == NULL`)

But this is a detail. The change looks good.